nixos/modules/services/web-apps/fittrackee.nix

```diff
@@ -0,0 +1,171 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.fittrackee;
+
+  commonService = {
+    wantedBy = [ "multi-user.target" ];
+    after = [
+      "network.target"
+      "postgresql.service"
+      "redis.service"
```

Comment:
If I understand correctly redis in nixos uses sub-instances (`services.redis.services`), shouldn't this module define a server dedicated to fit-trackee?